gatsby-browser.js

```diff
@@ -176,20 +176,7 @@ export const onClientEntry = () => {
     updateURLAsPerUserLanguage()
 }
 
-const scrollToAnchor = (location) => {
-    // Check for location so build does not fail
-    if (location && location.hash) {
-        window.scrollTo({
-            top: window.innerHeight,
-            behavior: 'smooth',
-        })
-    }
-
-    return true
-}
-
-export const onRouteUpdate = ({ location }) => {
-    scrollToAnchor(location)
+export const onRouteUpdate = () => {
```

Comment:
Do we need any changes in this file or we can make it same as master? @hasan-deriv 🙏

Reply:
Yeah, this code was the main problem.